documentation/pom.xml

```diff
@@ -22,7 +22,15 @@
     <description>The Hibernate Validator reference manual</description>
 
     <properties>
-        <jdocbook.ignoreTranslations>false</jdocbook.ignoreTranslations>
+        <asciidoctor.base-outputdir>${basedir}/target/asciidoctor</asciidoctor.base-outputdir>
+        <!--
+        With the asciidoctor Maven plugin 1.6, we are going to put back the images into the src/main/asciidoc
```

Comment:
OOI, what's the issue here that's fixed in that new version? A link to the issue in their tracker would be nice.